docs/tools/offline/cli-device-management.md

```diff
@@ -0,0 +1,86 @@
+### Updating Devices with Mbed CLI
+
+Mbed CLI includes features to prepare and ship updates for devices managed through the [Device Management Portal](https://cloud.mbed.com/docs/current/introduction/index.html). Mbed CLI uses the subcommands starting with `mbed device-management`, `mbed dev-mgmt` or `mbed dm` to manage devices.  As `device-management` is very long to type, the remainder of this document uses the `mbed dm` alias for all device management subcommands. This document explains the steps to enable and use Mbed Device Managment with a project.
```

Comment:
@bridadan That's what I had before @MelindaWeed Asked me to change it to what it is now.

Reply:
@MelindaWeed @AnotherButler is "Mbed Device Management" correct? Everything I've seen so far says wither "Pelion Device Management" or just "Device Management". Sorry, not trying to belabor this but since Mbed CLI is pretty forward facing it'd be worth it to get this one right. Also, I guess I want to know for myself going forward what the right phrasing is. Thanks!

Reply:
Sorry, I should've been more specific. 

'Mbed Device Management' is never used. Pelion Device Management is used at the first mention in a page or at the top level of any nested pages, but after that, the Pelion is dropped and it's just Device Management. 

Mbed CLI, Mbed OS, and Mbed client (but NOT Mbed *Cloud* client) keep their names "for brand recognition" 🙄

Reply:
This is indeed really confusing. Sorry for being unclear!

Reply:
Mbed is dropped entirely except in a certain few cases.

Reply:
No, Mbed CLI is still Mbed CLI.